File: introduction/ITP1_1_D-Watch.scala
object Main {
  def main(args:Array[String]) = {
    val sc = new java.util.Scanner(System.in)
    val time = sc.nextInt
    val hour = time / 3600;
    val minutes = (time / 60) % 60;
    val seconds = time % 60;
    println(hour + ":" + minutes + ":" + seconds)
  }
}

File: introduction/ITP1_1_D-Watch.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int time = Integer.parseInt(sc.nextLine());
    int hour = time / 3600;
    int minutes = (time / 60) % 60;
    int seconds = time % 60;
    System.out.println(hour + ":" + minutes + ":" + seconds);
  }
}
